src/vm/perfinfo.cpp

```diff
@@ -41,8 +41,21 @@ void PerfInfo::LogImage(PEFile* pFile, WCHAR* guid)
 
     SString value;
     const SString& path = pFile->GetPath();
-    PEImageLayout *pLoadedLayout = pFile->GetLoaded();
-    SIZE_T baseAddr = (SIZE_T)pLoadedLayout->GetBase();
+    if (path.IsEmpty())
+    {
+        return;
+    }
+
+    SIZE_T baseAddr = 0;
+    if (pFile->HasOpenedILimage())
```

Comment:
Should this rather use `IsILImageReadyToRun()` ? I assume we do not care about IL images here since they cannot contain any code.

Also, if the IL image does not contain any code, should we skip the logging altogether?

Reply:
I think both of these are good things to do.  I'll update the code to use `IsILImageReadyToRun()`.  Is there a cheap way to check if the image contains code?  I didn't see anything obvious on `PEImage`.

Reply:
If `IsILImageReadyToRun` returns true, it means the image contains code.

Reply:
Great.  I've pushed a fix for this.